sass/layout/_headers.scss

```diff
@@ -232,7 +232,7 @@
     }
 
     .channel-header__description {
-        height: 24px;
+        height: 21px;
```

Comment:
LIne height here is 20 so you see the tip of second line. Changing this close to line height to clip it